sklego/metrics.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def correlation_score(column):
+    """
+    The correlation score can score how well the estimator predictions
+    correlate with a given column. This is especially usefull to use
+    in situations where "fairness" is a theme.
+    :param column: Name of the column (when X is a dataframe)
+    or the index of the column (when X is a numpy array).
+    :return: Negative correlation between estimator.predict(X) and X[colum]
+    (in gridsearch, larger is better and we want to typically punish correlation).
+    """
+    def fairness_metric(estimator, X, y_true=None):
```

Comment:
Should we rename this to something indicating that it's a fairness metric only for regression tasks?

Reply:
*correlation_metric* for the win.